### ISSUE TITLE
数学公式自动标号和 \label 引用

### ISSUE THREAD

**Issue by 88250** [Vanessa219/vditor#713]
http://docs.mathjax.org/en/latest/input/tex/eqnumbers.html

**Comment by Vanessa219** [comment]
先关闭了，不太清楚是什么
